CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.1.1
+
+* Fix for encoding HTML for text string that contains `<pre>` [#263].(https://github.com/dart-lang/markdown/issues/263)
```

Comment:
You can use markdown here:

* Fix for encoding HTML for text string that contains `<pre>` ([#263](https://github.com/dart-lang/markdown/issues/263)).

Reply:
done.